### PR TITLE
Use a file to store the configuration to access the Nessus API

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request creates a file that contains information for interacting with Nessus on any instances that run the `nessus` Ansible role defined in this repository (the `vulnscan` instances in our Terraform configuration). The Python script that does some post-deployment configuration of Nessus is updated to use this file instead of having information templated into it by the Ansible role.

## 💭 Motivation and context ##

We are looking to make changes to jobs that are pushed to `vulnscan` instances that require the configuration information to be available on the instance somewhere. Since we need this information available it makes sense to store them in a static location on the instance and to update the configuration Python script to use this file.

## 🧪 Testing ##

Automated tests pass. I used this in my testing environment and verified that Nessus was configured as it should be post deployment and that the configuration file was on the instance.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
Nice improvement!

File: ansible/roles/nessus/tasks/main.yml [hunk 1]
```diff
@@ -78,28 +78,29 @@
...
+    group: cyhy
+    mode: 0750
+
+- name: Create the configuration file for Nessus API access
+  template:
+    src: nessus_api.yml.j2
+    dest: /etc/cyhy/nessus_api.yml
+    owner: cyhy
+    group: cyhy
+    mode: 0640
```
Comment (dav3r) [inline ansible/roles/nessus/tasks/main.yml:95]: Alphabetize?
```suggestion
  file:
    group: cyhy
    mode: 0750
    owner: cyhy
    path: /etc/cyhy
    state: directory

- name: Create the configuration file for Nessus API access
  template:
    dest: /etc/cyhy/nessus_api.yml
    group: cyhy
    mode: 0640
    owner: cyhy
    src: nessus_api.yml.j2
```

**Review by dav3r — APPROVED** [review]
Outstanding!  🥇 
I have but one minor quibble.

File: ansible/roles/nessus/tasks/main.yml (hunk 1 above)
Comment (mcdonnnj) [reply ansible/roles/nessus/tasks/main.yml:95]: I intentionally did not alphabetize to match the existing Ansible role structures and I was planning to update #478 once this PR is merged which would have those changes.

Comment (dav3r) [reply ansible/roles/nessus/tasks/main.yml:95]: OK, I suspected you had some motive here, but I wasn't sure what it was.  Thanks for the explanation. 👍

Comment (mcdonnnj) [reply ansible/roles/nessus/tasks/main.yml:95]: Just to circle back I have rebased #478 and the concerns raised here should be resolved in https://github.com/cisagov/cyhy_amis/pull/478/commits/984084dc5dc8056eda10b189ccf25ffd42b54141.

**Review by felddy — COMMENTED** [review]
Looks good!

**Pull request merged**